lib/icingadb/icingadb-objects.cpp

```diff
@@ -351,8 +544,6 @@ void IcingaDB::DeleteKeys(const std::vector<String>& keys, RedisConnection::Quer
 std::vector<String> IcingaDB::GetTypeObjectKeys(const String& type)
```

Comment:
Might be a good idea to rename this to reflect that these are actually the keys that are deleted and resynced from scratch with each sync.